.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release workflow
+
+on:
+  push:
+    branches: main
+
+env:
+  RUST_VERSION: rust:1.70.0
+
+jobs:
```

Comment:
I don't understand the rationale for this release job. First, there should be a separate job per product, not one big release thing for many different and unrelated things.

Second, if I ask it to release something I want it to do it, provided it will **not overwrite** anything. This is what we do for other releases in Jenkins also.

So this should 
- not be checking any changes
- it should not be doing automatic releases on pushes to main. If you really want this to be triggered automatically then it should be on creation of tags of the form `release/id-recover-tool/*.*.*-*` But then those tags must also be protected.
- it should push a properly version image, the version being the one found in Cargo.toml.
- should have a manual dispatch so I can trigger it when I want.

Reply:
Based on the inperson talk I will remove the build version from the release version.